templates/docs/api.html

```diff
@@ -4,11 +4,14 @@
 {% block twitter_title %}API{% endblock %}
 {% block og_title %}API{% endblock %}
 
+{% block title %}
+  <h1 id="maas-api">MAAS API</h1>
```

Comment:
```suggestion
  <h1 class="u-no-margin--bottom" id="maas-api">MAAS API</h1>
```
Looks like there a little too much space between heading and TOC